app/src/main/kotlin/it/fast4x/rimusic/ui/screens/player/PlayerModern.kt

```diff
@@ -2223,13 +2223,15 @@ fun PlayerModern(
                                 )
 
                     }
-                    if (!showthumbnail) {
-                        StatsForNerds(
-                            mediaId = mediaItem.mediaId,
-                            isDisplayed = statsfornerds,
-                            onDismiss = {}
-                        )
-                    }
+
+                    // does not seem to get activated so commented
+//                    if (!showthumbnail) {
+//                        StatsForNerds(
+//                            mediaId = mediaItem.mediaId,
+//                            isDisplayed = statsfornerds,
+//                            onDismiss = {}
+//                        )
+//                    }
```

Comment:
I don't think that this gets activated, I did not find when it gets called.

Reply:
This gets called when you are in landscape mode.

Reply:
I will create a new pull request as this code seems to be the wrong way.

Reply:
The app really has many options, such that we the devs have to test many scenarios.

Reply:
The other errors in horizontal have to be fixed sometime else.